ambry-server/src/main/java/com.github.ambry.server/AmbryRequests.java

```diff
@@ -806,6 +807,12 @@ private AdminResponse handleBlobStoreControlRequest(DataInputStream requestStrea
               if (storageManager.controlCompactionForBlobStore(partitionId, true)) {
                 error = ServerErrorCode.No_Error;
                 logger.info("Store is successfully started and functional for partition: {}", partitionId);
+                List<PartitionId> failToUpdateList =
+                    storageManager.setBlobStoreStoppedState(Arrays.asList(partitionId), false);
+                if (!failToUpdateList.isEmpty()) {
+                  logger.info("Fail to remove BlobStore(s) {} from stopped list after start operation completed",
```

Comment:
I don't think you need to do `Arrays.toString(failToUpdateList.toArray())`. Just adding the array as `failToUpdateList` is enough the logger should be able to resolve.

Also should this be `info` or `warn`? Seems like `warn` to me.

Reply:
Same down below

Reply:
done. Removed unnecessary `Arrays.toString`